src/components/common/NavBar.js

```diff
@@ -49,37 +54,42 @@ function NavBar(props) {
       >
         Home
       </Menu.Item>
-
-      <Menu.Item
-        key="2"
-        icon={<UserOutlined />}
-        onClick={() => history.push('/manage-members')}
-      >
-        Manage Members
-      </Menu.Item>
-
-      <Menu.Item
-        key="3"
-        icon={<CalendarOutlined />}
-        onClick={() => history.push('/manage-programs')}
-      >
-        Manage Programs
-      </Menu.Item>
-
-      <Menu.Item
-        key="4"
-        icon={<TeamOutlined />}
-        onClick={() => history.push('/manage-staff')}
-      >
-        Manage Staff
-      </Menu.Item>
-      <Menu.Item
-        key="5"
-        icon={<TeamOutlined />}
-        onClick={() => history.push('/manage-clubs')}
-      >
-        Manage Clubs
-      </Menu.Item>
+      {role === 'ADMIN' && (
```

Comment:
I think it should be ADMIN or CD instead of only ADMIN ?

Reply:
Makes sense.  I will add 'CD' too.